README.md

```diff
@@ -1,3 +1,5 @@
+> Note: This repository is archived for read-only reference purposes. Check out the [Reaction Component Library](https://github.com/reactioncommerce/reaction-component-library) for the replacement.
```

Comment:
what about the warning icon box like we're using in swag shop?

---

 **⚠  NOTICE:** This repository is archived for read-only reference purposes. Check out the [Reaction Component Library](https://github.com/reactioncommerce/reaction-component-library) for the replacement.

---